File: Codeforces/A/236A.cpp
#include <bits/stdc++.h>
 
using namespace std;
 
int main()
{
    string input; bool letters[26]; int counter = 0;
    for ( int i = 0; i < 26; ++i )
        letters[i] = false;
 
    getline(cin, input);
    int n = input.size();
 
    for ( int i = 0; i < n; ++i)
    {
        if ( letters[input[i] - 'a'] == false )
        {
            letters[input[i] - 'a'] = true;
            counter++;
        }
    }
 
    if ( counter % 2 == 1 ) cout << "IGNORE HIM!";
    else cout << "CHAT WITH HER!";
 
    return 0;
}

File: Codeforces/A/236A.cs
using System;
 
namespace MainProgram
{
    class Program
    {
        static void Main(string[] args)
        {
            string input = Console.ReadLine();
            System.Collections.ArrayList dElements = new System.Collections.ArrayList();
            for (int i = 0; i < input.Length; i++)
                if (!dElements.Contains(input[i])) dElements.Add(input[i]);
            if(dElements.Count % 2 == 0)
                Console.WriteLine("CHAT WITH HER!");
            else
                Console.WriteLine("IGNORE HIM!");
        }
    }
}